builder/github.go

```diff
@@ -12,14 +13,17 @@ type GitHubClient interface {
 
 // NewGitHubClient returns a new GitHubClient instance. If token is an empty
 // string, then a fake client will be returned.
-func NewGitHubClient(token string) GitHubClient {
+func NewGitHubClient(token string, domain string) GitHubClient {
 	if token == "" {
 		return &nullGitHubClient{}
 	}
 
 	ts := oauth2.StaticTokenSource(&oauth2.Token{AccessToken: token})
 	tc := oauth2.NewClient(oauth2.NoContext, ts)
-	return github.NewClient(tc).Repositories
+	c := github.NewClient(tc)
+	c.BaseURL, err = url.Parse(domain)
```

Comment:
This shouldn't run if `domain` is an empty string, so that we default to api.github.com. Also, we shouldn't ignore the error here. A `if err != nil { panic(err) }` would be fine.